tests/functional/aws-node-sdk/test/multipleBackend/delete/deleteAwsVersioning.js

```diff
@@ -557,3 +583,83 @@ describeSkipIfNotMultiple('AWS backend delete object w. versioning: ' +
         });
     });
 });
+
+
+describeSkipIfNotMultiple('AWS backend delete multiple objects w. versioning: ' +
+    'using object location constraint', function testSuite() {
+    this.timeout(120000);
+    withV4(sigCfg => {
+        let bucketUtil;
+        let s3;
+        beforeEach(() => {
+            process.stdout.write('Creating bucket\n');
+            bucketUtil = new BucketUtility('default', sigCfg);
+            s3 = bucketUtil.s3;
+            return s3.createBucket({ Bucket: bucket }).promise()
```

Comment:
you could use async await here btw, quite minor but you probably wouldn't need to catch the error just to throw it back. If you let the test runner catch the error the call stack is already available.